server/routers/billingRouter.js

```diff
@@ -1,24 +1,153 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const billing = require('../data/helpers/billingModel')
-
-
+const billing = require("../data/helpers/billingModel");
 
 /************************************ BILLING SECTION ***********************************/
 
 /********* Get Billing *************/
-router.get('/', async (req, res) => {
-  await  billing.get()
-        .then((bills) => {
-            res.json(bills);
+router.get("/", async (req, res) => {
+  await billing
+    .get()
+    .then(bills => {
+      res.json(bills);
+    })
+    .catch(err => {
+      res.status(500).json({ error: "The billing could not be retrieved." });
+    });
+});
+
+/********* Get Single Bill *************/
+router.get("/:id", async (req, res) => {
+  const { id } = req.params;
+  await billing
+    .get(id)
+    .then(bills => {
+      if (bills) {
+        res.json(bills);
+      } else {
+        res
+          .status(404)
+          .json({ message: "The bill with the specified ID does not exist." });
+      }
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: "The bill information could not be retrieved." });
+    });
+});
+
+/************* Delete Bill *************/
+router.delete("/:id", (req, res) => {
+  const { id } = req.params;
+
+  if (id) {
+    billing
+      .remove(id)
+      .then(bills => {
+        if (bills) {
+          res.json({ message: "The bill was successfully deleted" });
+        } else {
+          res.status(404).json({
+            message: "The bill with the specified ID does not exist."
+          });
+        }
+      })
+      .catch(err => {
+        res.status(500).json({ error: "The bill could not be removed." });
+      });
+  }
+});
+
+/********* Update Billing *************/
+router.put("/:id", (req, res) => {
+  const { id } = req.params;
+  const newBills = req.body;
+  if (
```

Comment:
I agree it could be better to create a object with them all and run that through a for loop or map. And I am not sure what we are going to be updating

Reply:
I would suggest that we leave as is for now until we understand how the information from stripe is going to feed into this.